### PR TITLE
Twemoji

### PR DESCRIPTION
**Pull request by dbrgn**
Switch from Emojione 3 to current Twemoji (by Twitter).

This adds Unicode 11.0 Emoji.

img

### CODE REVIEW

File: public/emoji/activity.css [hunk 1]
```diff
@@ -0,0 +1,1933 @@
+/* glue: 0.13 hash: efcf7f4b00 */
+        .em-activity-26f9,
```
Comment (lgrahl) [inline public/emoji/activity.css]: Nit: Indentation intentional?

File: public/emoji/activity.css [hunk 2]
```diff
@@ -0,0 +1,1933 @@
...
+        }
+        
+        .em-activity-1f396 {
+            background-position: -511px -477px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        @media screen and (-webkit-min-device-pixel-ratio: 1.0), screen and (min--moz-device-pixel-ratio: 1.0), screen and (-o-min-device-pixel-ratio: 100/100), screen and (min-device-pixel-ratio: 1.0), screen and (min-resolution: 1.0dppx) {
+            .em-activity-26f9,
```
Comment (lgrahl) [inline public/emoji/activity.css]: Same here.

File: public/emoji/flags.css [hunk 3]
```diff
@@ -0,0 +1,2069 @@
+/* glue: 0.13 hash: 69dcc6a805 */
+        .em-flags-1f6a9,
```
Comment (lgrahl) [inline public/emoji/flags.css]: ^

File: public/emoji/flags.css [hunk 4]
```diff
@@ -0,0 +1,2069 @@
...
+        }
+        
+        .em-flags-1f1e6-1f1e9 {
+            background-position: -545px -1px;
+            width: 32px;
+            height: 32px;
+        }
+        
+        @media screen and (-webkit-min-device-pixel-ratio: 1.0), screen and (min--moz-device-pixel-ratio: 1.0), screen and (-o-min-device-pixel-ratio: 100/100), screen and (min-device-pixel-ratio: 1.0), screen and (min-resolution: 1.0dppx) {
+            .em-flags-1f6a9,
```
Comment (lgrahl) [inline public/emoji/flags.css]: ^

File: public/emoji/food.css [hunk 5]
```diff
@@ -0,0 +1,853 @@
+/* glue: 0.13 hash: 31946c9658 */
+        .em-food-2615,
```
Comment (lgrahl) [inline public/emoji/food.css]: and so on...

File: public/emoji/activity.css (hunk 1 above)
Comment (dbrgn) [reply public/emoji/activity.css]: This is a generated file :slightly_smiling_face:

File: public/emoji/activity.css (hunk 2 above)
Comment (dbrgn) [reply public/emoji/activity.css]: This is a generated file :slightly_smiling_face:

File: public/emoji/flags.css (hunk 4 above)
Comment (dbrgn) [reply public/emoji/flags.css]: This is a generated file :slightly_smiling_face:

File: public/emoji/food.css (hunk 5 above)
Comment (dbrgn) [reply public/emoji/food.css]: This is a generated file :slightly_smiling_face:

File: public/emoji/flags.css (hunk 3 above)
Comment (dbrgn) [reply public/emoji/flags.css]: This is a generated file :slightly_smiling_face:

**Pull request merged**